### PR TITLE
add create table cg_trail_history to onCreate (fix #7482)

### LINKED ISSUE #7482: c:geo crashes with "no such table: cg_trail_history" after fresh install
##### Detailed steps causing the problem:
- fresh install of current (April 10) master brach on a new device
- run c:geo and open Map

##### Actual behavior after performing these steps:
- c:geo crashes with log message "no such table: cg_trail_history"

##### Expected behavior after performing these steps:
- show map

##### Version of c:geo used:
- today's (April 10) master

##### Is the problem reproducible for you?
Yes

##### System information:
- tested under API 21 (Android 5.0) with Google API enabled

##### Other comments and remarks:
- table "cg_trail_history" got introduced by fix for #2958
- why is the database update routine not called before opening the map / what has to be done that it gets called before?

### PR DESCRIPTION
**Pull request by moving-bits**

### CODE REVIEW

File: main/src/cgeo/geocaching/storage/DataStore.java [hunk 1]
```diff
@@ -560,7 +560,7 @@ public void onCreate(final SQLiteDatabase db) {
             db.execSQL(dbCreateLogsOffline);
             db.execSQL(dbCreateTrackables);
             db.execSQL(dbCreateSearchDestinationHistory);
-
+            db.execSQL(dbCreateTrailHistory);
```
Comment (rsudev) [inline main/src/cgeo/geocaching/storage/DataStore.java]: Only a minor thing, but I would keep the blank line between the table creation statements and the index call.

**Comment by Lineflyer** [comment]
retest this please

**Comment by Bananeweizen** [comment]
This is an issue for further PRs: Please use the "fix #1234" notation already in the commit message, not only in the title of the pull request. Example from samuel: https://github.com/cgeo/cgeo/pull/7455/commits
If you do that, github automatically links back to the original issue (while it is hard to navigate to the issues of your PRs), and autoclosing may also not work for your PRs: https://help.github.com/en/articles/closing-issues-using-keywords

**Comment by Lineflyer** [comment]
Just for completeness:
Autoclose also works if the "fix #1234" is contained in the PR name, however I agree with the better hyperlinking if it is referenced (also) in the commit message, so we should use it like that.

File: main/src/cgeo/geocaching/storage/DataStore.java (hunk 1 above)
Comment (moving-bits) [reply main/src/cgeo/geocaching/storage/DataStore.java]: done

**Pull request merged**